backend/src/database/config.ts

```diff
@@ -0,0 +1,15 @@
+import '../setup'
+
+import { dbConfig } from './util'
+
+const commonConfig = {
+  dialect: 'postgres',
+  seederStorage: 'sequelize',
+  ...dbConfig,
```

Comment:
```suggestion
migrationStorageTableName: 'sequelize_meta',
seederStorageTableName: 'sequelize_data',
  ...dbConfig,
```

Might want to rename to underscore case to match our other table names.